packages/ui/src/views/repo/pull-request/details/components/conversation/pull-request-status-select-button.tsx

```diff
@@ -18,17 +18,14 @@ interface StatusButtonProps {
   onChange: (value: string) => void
 }
 
-const StatusButton: React.FC<StatusButtonProps> = memo(({ codeCommentStatus, onChange }) => (
-  <Select placeholder="Status" onValueChange={onChange}>
-    <SelectTrigger>
-      <SelectValue className="py-2" placeholder={codeCommentStatus} />
-    </SelectTrigger>
+const StatusButton: React.FC<StatusButtonProps> = ({ codeCommentStatus, onChange }) => (
+  <Select placeholder={codeCommentStatus} onValueChange={onChange}>
     <SelectContent>
       <SelectItem value={CodeCommentState.ACTIVE}>Active</SelectItem>
       <SelectItem value={CodeCommentState.RESOLVED}>Resolved</SelectItem>
     </SelectContent>
   </Select>
-))
+)
 StatusButton.displayName = 'StatusButton'
 
 export const PullRequestStatusSelect: React.FC<CodeCommentStatusSelectProps> = ({
```

Comment:
See if we can avoid using React.FC for all instances in your changes.

Reply:
done